src/_pytest/terminal.py

```diff
@@ -1375,7 +1375,7 @@ def _build_normal_summary_stats_line(
                 count = len(reports)
                 color = _color_for_type.get(key, _color_for_type_default)
                 markup = {color: True, "bold": color == main_color}
-                parts.append(("%d %s" % pluralize(count, key), markup))
+                parts.append(("%d %s" % pluralize(count, key), markup))  # noqa: UP031
```

Comment:
I noqa'd three such constructs, but we could extract the variable from pluralize too, or drop the %d

Reply:
That's fine. 👍